Assets/AishaBikebayeva/Arcade Game/Scripts/gameOverScreen.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class gameOverScreen : MonoBehaviour
```

Comment:
Capitalization, should be `GameOverScreen` for good practice.

Also missing namespace